src/vcWebFile.cpp

```diff
@@ -64,20 +64,23 @@ udResult vcWebFile_Open(udFile **ppFile, const char *pFilename, udFileOpenFlags
   pFile->fpRead = vcWebFile_SeekRead;
   pFile->fpClose = vcWebFile_Close;
 
-  UD_ERROR_IF(vdkWeb_RequestAdv(pFilename, options, &pData, &dataLength, &responseCode) != vE_Success, udR_OpenFailure);
-
-  // TODO: (EVC-615) JIRA task to expand these
-  if (responseCode == 403)
-    UD_ERROR_SET(udR_NotAllowed);
-  else if (responseCode == 503)
-    UD_ERROR_SET(udR_Pending);
-  else if (responseCode >= 500 && responseCode <= 599)
-    UD_ERROR_SET(udR_ServerError);
-  else if (responseCode < 200 || responseCode >= 300)
-    UD_ERROR_SET(udR_OpenFailure);
-
-  pFile->totalBytes = dataLength;
-  pFile->fileLength = dataLength;
+  if (!(flags & udFOF_FastOpen))
```

Comment:
I feel like we've looked into this before and it created problems, does this just work? How much testing have you done on this? I think the tile server was really one of the only places this was used.

Reply:
Yes, with the change to make vcTileRenderer use this it does appear to work

Reply:
Not much testing though, only really ran it for like 10 minutes